content/ui-patterns/messaging.mdx

```diff
@@ -0,0 +1,148 @@
+- _Flash alerts are best suited for this type of messaging._
+
+## Toasts
+<Box mb="3">
+  <img width="453" src="https://user-images.githubusercontent.com/586552/63105014-f5477a00-bf4d-11e9-954e-f01582a39ef3.png"/>
+</Box>
+
+[Toasts](https://primer.style/css/components/toasts) are used to provide relevant feedback to the user, typically after they've taken an action. Primer includes styles for success, warning, error, loading, and default Toasts. Some examples include:
+
+- confirmation that an action was successfully taken
+- communicating that an action is pending
+- notifying the user if an action failed to complete.
+- providing general information.
+
+Toasts are best used in context of the page they're referring to (rather than a global notification that can appear on any page) and should require minimal user interaction. For global notices and messaging, see the [Flash component](#flash-alerts).
+
+Toasts should be brief and not bog down the experience with superfluous copy. If multiple toasts appear on the page, they will stack naturally.
+
+<Flex flexDirection={['column', 'row', 'row']}>
+  <Box mr="2"  mb="4">
+    <Flex flexDirection="row" alignItems="center" mb="2">
+      <CircleOcticon icon={Check} size={16} bg="green.5" color="white" mr="2" p="1"/>
+      <Text fontWeight="bold" color="gray.9" ml="1">
+        Do
+      </Text> 
+    </Flex>
+    <Box>
+      <img src="https://user-images.githubusercontent.com/586552/63106528-06de5100-bf51-11e9-8a5e-98583ed74874.png" />
+    </Box>
+    <Text fontSize="0" color="gray.5">
+      Use brief and direct communication
+    </Text>
+  </Box>
+  <Box mr="2">
+    <Flex flexDirection="row" alignItems="center" mb="2">
+      <CircleOcticon icon={X} size={16} bg="red.5" color="white" mr="2" p="1"/>
+      <Text fontWeight="bold" color="gray.9" ml="1">
+        Don't
+      </Text> 
+    </Flex>
+    <Box borderRadius="3">
+      <img src="https://user-images.githubusercontent.com/586552/63106527-06de5100-bf51-11e9-858c-72de6a5c728a.png" />
+    </Box>
+    <Text fontSize="0" color="gray.5">
+      Don't use wordy and redundant copy
+    </Text>
+  </Box>
+</Flex>
+
+### Accessibility
+Toasts are non-modal components and should contain `role=log` , which implies the element has `aria-live="polite"`. This notifies the user of the Toast via Assistance Technologies without having to change focus to the Toast. You can read more about `role=log` [here](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA23).
+
+Toasts are generally informative and should not receive focus when they appear. For that reason, we suggest you **avoid using interactive elements** in the component (aside from a dismiss action). Keep in mind that, even without an explicit dismiss action, the user can always hit `esc` to dismiss the Toast. For more information on how interactive children affect web accessibility, [check out this issue](https://github.com/jackbsteinberg/std-toast/issues/29).
```

Comment:
```suggestion
Toasts are generally informative and should not receive focus when they appear. For that reason, we suggest you **avoid using interactive elements** in the component (aside from a dismiss action). Keep in mind that, even without an explicit dismiss action, the user can always hit `esc` to dismiss the toast. For more information on how interactive children affect web accessibility, [check out this issue](https://github.com/jackbsteinberg/std-toast/issues/29).
```

Reply:
This should remain capitalized, since it's the name of a component.